mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -1,18 +1,21 @@
 package org.mlflow.tracking;
 
+import com.google.common.collect.Lists;
 import org.apache.http.client.utils.URIBuilder;
-
+import org.mlflow.api.proto.ModelRegistry.*;
 import org.mlflow.api.proto.Service.*;
+import org.mlflow.artifacts.CliBasedArtifactRepository;
+import org.mlflow.tracking.creds.*;
```

Comment:
nit: move imports to the alphabetically correct position

Reply:
Done